syntax = "proto3";

package contract_checker;


message ApprovalState {
    bool state = 1;
    string person = 2;
}

message Approval {
    ApprovalState planning = 1;
    ApprovalState legal = 2;
    ApprovalState sales = 3;
    ApprovalState manager = 4;
}
